pages/index.html

```diff
@@ -1,12 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+  <script src="https://kit.fontawesome.com/0dbf060281.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Marck+Script&family=Noto+Serif:ital,wght@0,400;0,700;1,400;1,700&family=Rubik+Distressed&family=Rubik+Microbe&family=Shadows+Into+Light+Two&display=swap"
+    rel="stylesheet">
+  <title>Vivian's Home Page</title>
 </head>
+
 <body>
-  
+  <header>
+    <p>Vivian Zhu's Lab</p>
+  </header>
+
+  <div class="nav_bar">
+    <a class="active" href="index.html"><i class="fa fa-light fa-house"></i> Home </a>
+    <a href="about.html"><i class="fa fa-fw fa-person"></i> About </a>
+    <a href="portfolio.html"><i class="fa fa-fw fa-book"></i> Portfolio </a>
+    <a href="talks.html"><i class="fa-solid fa-microphone"></i> Talks </a>
+  </div>
+
+  <div class="description">
+    <article>
+      <h2>Welcome to my Lab!</h2>
+      <p><i class="fa fa-thin fa-door-open fa-beat"></i>
+      <i class="fa-solid fa-flask-vial fa-shake"></i></p>
+      <p>Life is a lab full of experiments.</p>
+      <p>There will be milestones and celebrations</p>
+      <p>There will also be setbacks and regrets</p>
+      <p>In this experimental space</p>
+      <p>I hope to find solutions and embrace the potential.</p>
+
+
+    </article>
+  </div>
+
+  <div class="connect_info">
```

Comment:
When using semantic HTML I recommend trying to use semantic tags as much as possible - something like `<aside>` might work well here.